packages/melange.ppx/ppx.ml

```diff
@@ -765,6 +765,7 @@ class raise_exception_mapper (module_path : string) =
     inherit Ast_traverse.map as super
```

Comment:
This traverse is called `validate_record_labels`, but it works on `mel.*` extensions. This name seems weird to me because the context is applied.